app/src/main/java/com/kickstarter/viewmodels/CreatorDashboardFragmentViewModel.java

```diff
@@ -51,6 +51,10 @@ public ViewModel(final @NonNull Environment environment) {
         .compose(bindToLifecycle())
         .subscribe(this.projectAndStats);
 
+      this.projectsMenuClick
```

Comment:
Could we rename this to `projectList` instead? It would make more sense on the context of the app.